tests/profiles/test_base_profile.py

```diff
@@ -36,17 +36,58 @@ def test_validate_profile_args(profile_arg: str):
         )
 
 
-@pytest.mark.parametrize("disable_event_tracking", [True, False])
-def test_disable_event_tracking(disable_event_tracking: str):
+@pytest.mark.parametrize("dbt_config_var,dbt_config_value", [("debug", True), ("debug", False)])
+def test_validate_dbt_config_vars(dbt_config_var: str, dbt_config_value: Any):
```

Comment:
this test should be kept to validate that `disable_event_tracking` still works as expected and you can add the new tests below.

Reply:
return it)